mscore/revision.h

```diff
@@ -1 +1 @@
-3543170
+c1a89a0
```

Comment:
Don't commit changes to this file, ever.
In your clone directory, copy the file build/git/hooks/post-checkout to the directory .git/hooks in order for mscore/revision.h to be maintained automatically by git. See build/git/hooks/README for details. Note that the .git directory is hidden.